README.md

```diff
@@ -52,7 +52,10 @@ There are numerous resources that describe the quantum Fourier transformation an
 including:
 
  * Several sections in the [Qiskit textbook](https://qiskit.org/textbook)
-
```

Comment:
You added these references to the "sections in the Qiskit textbook" list, but they are not sections in the Qiskit textbook, they are separate papers and blogs. I will merge and fix.